### DISCUSSION TITLE
How to trig a notification for another user

### DISCUSSION THREAD

**Discussion by KONEY** [filamentphp/filament#6658] [Help]
I'm not sure if this is possible at all but I'd like to ask: is there a way to trig a popup notification into another user session?

thanks!

**Comment by danharrin** [comment]
You would need to use a database or broadcast notification.

**Reply by KONEY** [reply]
Thanks @danharrin! So it is possible to deliver notifications among authenticated users...?

**Reply by danharrin** [reply]
Yes

**Reply by KONEY** [reply]
Cool!
So accordingly to docs this should save and send a notification to user with id 1:
`$user=\App\Models\User::find(1);`
`Notification::make()->title('Nuovo messaggio in chat 1!')->success()->sendToDatabase($user);`
`event(new DatabaseNotificationsSent($user));`

a record is created in the notifications table but no notification is displayed.

Instead this triggers one but only for the actual user:

`Notification::make()->title('Nuovo messaggio in chat 2!')->success()->send();`

There's not much else on the docs so any hint would be helpful!

Thanks

**Reply by KONEY** [reply]
So after some discussion in the docs area I figured out this question is not precise because what I wanted to do specifically is trigger a "flash notification" for another user. @danharrin can you confirm this can't be done?

Thanks

**Reply by danharrin** [reply]
A broadcast notification is very similar to a flash notification and can be sent to another user
